phpunit.xml.dist

```diff
@@ -1,23 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.8/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/7.0/phpunit.xsd"
          backupGlobals="false"
          colors="true"
          bootstrap="vendor/autoload.php">
   <testsuites>
     <testsuite name="Datanova-bundle test suite">
-      <directory>./Tests</directory>
+      <directory>./test</directory>
```

Comment:
not sure about the directory structure when I read the official documentation recommendations https://symfony.com/doc/current/bundles/best_practices.html#directory-structure